modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/FillExtrusionMBLayer.java

```diff
@@ -278,17 +278,17 @@ public FeatureTypeStyle transformInternal(MBStyle styleContext) {
         //setLegend(null) to empty list.
         rule.setLegendGraphic(new Graphic[0]);
 
-
-        return sf.featureTypeStyle(
-                getId(),
-                sf.description(
-                        Text.text("MBStyle "+getId()),
-                        Text.text("Generated for "+getSourceLayer())),
-                null, // (unused)
-                Collections.emptySet(),
-                filter.semanticTypeIdentifiers(),
-                Arrays.asList(rule)
-        );
+        return null;
```

Comment:
If we aren't returning anything here, we should remove the whole method rather than just the return statement.

We are probably fine just returning a fill for now?